### PR TITLE
added sac_act_repeat docs with GIFs and added the .gin files.

### PR DESCRIPTION
**Pull request by jesbu1**

### CODE REVIEW

File: alf/environments/suite_robotics.py [hunk 1]
```diff
@@ -75,8 +76,8 @@ def step(self, action):
         self._steps += 1
 
         info["success"] = 0.0
-        # only count success at the episode end
-        if self._steps == self._max_episode_steps and info["is_success"] == 1:
+        # only count success after a certain amount of steps
+        if self._steps == self._since_episode_steps and info["is_success"] == 1:
```
Comment (hnyu) [inline alf/environments/suite_robotics.py]: == should be >=?

File: docs/notes/fetch_env_sac_actrepeat_notes.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
+Fetch Environments with SAC + Action Repeat
+===========================================
+
+By using regular Soft Actor Critic (SAC) with an Action Repeat length of 3,
+we are able to achieve very good performance on the Fetch environments
+introduced here: https://openai.com/blog/ingredients-for-robotics-research/
+
+The performance/sample efficiency is better DDPG+HER (Hindsight Experience
```
Comment (hnyu) [inline docs/notes/fetch_env_sac_actrepeat_notes.rst]: better than

File: docs/notes/fetch_env_sac_actrepeat_notes.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+Replay) in some environments as in their report:
+https://arxiv.org/pdf/1802.09464.pdf, despite the lack of HER in our
+implementation.
+
+We have 38 parallel rollouts and run for 4.75M timesteps total, just like in
```
Comment (hnyu) [inline docs/notes/fetch_env_sac_actrepeat_notes.rst]: in total

File: docs/notes/fetch_env_sac_actrepeat_notes.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+the Fetch report. We performed a search around the ``unroll_length``, which
+dictates the frequency at which we train our policy and Q functions. We found
+that an unroll length of 8 ``(50 // [3 (Action repeat length) * 2])``, which
+essentially doubles the number of training steps as in the report, is the
+best for both sample efficiency and final performance. All other parameters
+were chosen to match those in the report as closely as possible, although we
+do not incorporate an action L2 norm penalty.
```
Comment (hnyu) [inline docs/notes/fetch_env_sac_actrepeat_notes.rst]: ... and observation normalization.

File: docs/notes/fetch_env_sac_actrepeat_notes.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+**Note that due to the Action Repeat length of 3, the timesteps below should be multiplied by 3 for the real environment steps taken**
```
Comment (hnyu) [inline docs/notes/fetch_env_sac_actrepeat_notes.rst]: ... and the videos have a x3 speedup.

File: docs/notes/fetch_env_sac_actrepeat_notes.rst [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+The performance/sample efficiency is better DDPG+HER (Hindsight Experience
+Replay) in some environments as in their report:
+https://arxiv.org/pdf/1802.09464.pdf, despite the lack of HER in our
+implementation.
+
+We have 38 parallel rollouts and run for 4.75M timesteps total, just like in
+the Fetch report. We performed a search around the ``unroll_length``, which
+dictates the frequency at which we train our policy and Q functions. We found
+that an unroll length of 8 ``(50 // [3 (Action repeat length) * 2])``, which
```
Comment (hnyu) [inline docs/notes/fetch_env_sac_actrepeat_notes.rst]: explain that 50 is the episode length

**Review by hnyu — CHANGES_REQUESTED** [review]
Thanks for the PR and it looks great. Some minor comments

**Comment by hnyu** [comment]
Maybe also report the wall-clock training time of a run?

File: docs/notes/fetch_env_sac_actrepeat_notes.rst [hunk 7]
```diff
@@ -0,0 +1,53 @@
+Fetch Environments with SAC + Action Repeat
```
Comment (hnyu) [inline docs/notes/fetch_env_sac_actrepeat_notes.rst:1]: Should add this file as one entry in "docs/notes.rst". And probably also add a link pointing to this note file in README.md under "SAC" section.

**Comment by jesbu1** [comment]
Probably can't report accurate walltime numbers because I ran it while concurrently running 7 kickball searches.

**Review by hnyu — APPROVED** [review]

**Pull request merged**